just a message

```diff
@@ -0,0 +1,19 @@
+Hey Ian!
+Sorry for contacting you in this way, but I haven't found another workaround.
+
+Thank you a lot for composing the article about randomization <> analysis unit
+It makes a great job of describing the problem in a very intuitive way.
+I wanted to share it with my data science colleagues, so they can learn about it,
+but I think there's a critical typo in the variance formula in the delta method - 
+The first term should be Var(S)/N^2, not Var(S)/N - this is stated in the original paper:
+page 3 formula 4) and it aligns with my calculations.
```

Comment:
hey @AlexBogs , thanks for bringing this to my attention. you are completely correct this is a typo - i implemented the correct version in [the code](https://gist.github.com/ian-whitestone/84fd815e272df7f026791c9cd4612e9f):

![CleanShot 2021-11-26 at 14 04 46@2x](https://user-images.githubusercontent.com/11096727/143622125-30c38e7c-e22f-4167-a606-547da109e905.png)

Reply:
Thank you for the swift turnaround!